spec/requests/event_spec.rb

```diff
@@ -33,17 +33,16 @@
       end
     end
 
-    describe "logged in" do
+    describe "logged in and speaker_entry is enabled" do
```

Comment:
今 rspec の出力はあまり使用していないので重要な問題ではないですが、 `describe` は基本的にはテストのターゲットを指定すべきで、条件については `context` で記述した方が良さそうです。
```suggestion
    context "when logged in and speaker_entry is enabled" do
       context "when not registerd" do
       ...
       end
     end
```

Reply:
なるほどー

Reply:
Rspec の過剰な運用上の規約という感じもありますが、describe だと `described_class` が example で使用できるとか it 内で subject を使用すると Rspec の出力に自動で命名してくれるとかちょっとしたお得機能がつかえるようになりますね（Rspec そんなに使ったことが無いのですが）。